src/Net/AmqpSettings.cs

```diff
@@ -70,7 +70,7 @@ public int MaxLinksPerSession
         /// <summary>
         /// Gets or sets the open.idle-time-out field.
         /// </summary>
-        public int IdleTimeout
+        public uint IdleTimeout
```

Comment:
A property type change is potentially a breaking change so let's keep it as int. In OnOpen there is max value check so it should be able to handle any received idle timeout value.